temperature/schemas.py

```diff
@@ -0,0 +1,13 @@
+import datetime
+
+from pydantic import BaseModel
+
+
+class Temperature(BaseModel):
+    id: int
+    city_name: str
```

Comment:
The `city_name` field is included in the `Temperature` schema, but according to the task description, the `Temperature` model should have a `city_id` field instead. Ensure that the schema aligns with the database model and task requirements.